src/lib/components/Segment.svelte

```diff
@@ -114,10 +129,19 @@
       -1px,
       0
     );
-    width: calc((100% - (var(--segments) * var(--padding))) / var(--segments));
-    padding: var(--padding-2x) 0 var(--padding);
+    width: calc(
+      (
+        (
+            100% -
```

Comment:
This indentation here looks quite random?

Reply:
Extra parenthesises were removed 👍